config/itemTypes.json

```diff
@@ -1,136 +1,139 @@
 [{
-  "id": "Archwing",
+  "id": "/Archwing",
   "name": "Archwing"
 }, {
-  "id": "Powersuits",
+  "id": "/Powersuits",
   "name": "Warframe"
 }, {
-  "id": "Projections",
+  "id": "/Projections",
   "name": "Relic"
 }, {
-  "id": "Restorative",
+  "id": "/Restorative",
   "name": "Gear"
 }, {
-  "id": "Keys",
+  "id": "/Keys",
   "name": "Key"
 }, {
-  "id": "Sigils",
+  "id": "/Sigils",
   "name": "Sigil"
 }, {
-  "id": "Skins",
+  "id": "/Skins",
   "name": "Skin"
 }, {
-  "id": "ShipDecos",
+  "id": "/ShipDecos",
   "name": "Ship Decoration"
 }, {
-  "id": "ShipFeatureItems",
+  "id": "/ShipFeatureItems",
   "name": "Ship Segment"
 }, {
-  "id": "Ships",
+  "id": "/Ships",
   "name": "Orbiter"
 }, {
-  "id": "Ship",
+  "id": "/Ship",
   "name": "Extractor"
 }, {
-  "id": "CatbrowPet",
+  "id": "KubrowPetPowerSuit",
```

Comment:
does `PetPowerSuit` match both?

Reply:
Helios also has PetPowerSuit in the unqueName.

Reply:
nvm, it also matches something for helios